assets/css/properties_panel/_vehicle_properties_panel.scss

```diff
@@ -41,9 +41,13 @@
   margin-bottom: 0.25rem;
 }
 
-.c-vehicle-properties-panel__link {
-  @include font-link;
-  margin-bottom: 0.25rem;
+.c-vehicle-properties-panel__directions {
+  @include button-primary;
+  @include button-icon(1rem);
+  display: inline-flex;
+  align-items: center;
+  gap: 4px;
```

Comment:
**thought:** I know `gap` is a new-ish CSS feature, it might be worth comparing [the browser compatibility info from caniuse](https://caniuse.com/?search=gap) against our supported browser policy, if you haven't already.

Reply:
Good call - just checked and our supported browser versions all have support for it!